vimode/src/utils.c

```diff
@@ -219,3 +220,12 @@ void goto_nonempty(ScintillaObject *sci, gint line, gboolean scroll)
 		pos = NEXT(sci, pos);
 	SET_POS(sci, pos, scroll);
 }
+
+
+gboolean is_start_of_line(ScintillaObject *sci, gint pos)
```

Comment:
This function can go directly above the function where it's used and be static - there doesn't seem to be any need for it in the rest of the code.